08-projects.Rmd

```diff
@@ -4010,7 +4010,9 @@ An example of the data for the year 1987 can be found [here](https://www.datadep
 
 ##### 1. Let's say we have a theory that there are more flights on the weekend days (Friday, Saturday, Sunday) than the rest of the days, on average. We can use awk to quickly check it out and see if maybe this looks like something that is true!
 
-##### Write a line of `awk` code that, prints the total number of flights that occur on weekend days, followed by the total number of flights that occur on the weekdays. Complete this calculation for 2008 using the `2008.csv` file.
+##### Write a line of `awk` code that, prints the _total number of flights_ that occur on weekend days, followed by the _total number of flights_ that occur on the weekdays. Complete this calculation for 2008 using the `2008.csv` file.  
```

Comment:
In the first paragraph of Question 1, there is the phrase "on average". I italicize the two phrases here for clarification.